samples/websocket/chatApp/bundle/resources/pom.xml

```diff
@@ -0,0 +1,437 @@
+<!--
+  ~   Copyright (c) ${date}, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+  ~
+  ~   WSO2 Inc. licenses this file to you under the Apache License,
+  ~   Version 2.0 (the "License"); you may not use this file except
+  ~   in compliance with the License.
+  ~   You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~  Unless required by applicable law or agreed to in writing,
+  ~  software distributed under the License is distributed on an
+  ~  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  ~  KIND, either express or implied.  See the License for the
+  ~  specific language governing permissions and limitations
+  ~  under the License.
+  -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>org.wso2.sample</groupId>
+    <artifactId>sample-feature-installation</artifactId>
+    <version>1.0.0</version>
+    <packaging>pom</packaging>
+    <name>New feature</name>
+
+    <properties>
+        <carbon.messaging.version>1.0.12-SNAPSHOT</carbon.messaging.version>
+        <carbon.transport.http.netty.version>3.0.4-SNAPSHOT</carbon.transport.http.netty.version>
+        <carbon.metrics.version>2.0.0</carbon.metrics.version>
+        <msf4j.version>2.1.1-SNAPSHOT</msf4j.version>
+        <carbon.feature.plugin.version>2.1.0</carbon.feature.plugin.version>
+        <carbon.kernel.version>5.1.0</carbon.kernel.version>
+        <osgi.api.version>6.0.0</osgi.api.version>
+        <carbon.jndi.version>1.0.0</carbon.jndi.version>
+        <carbon.cache.version>1.0.0</carbon.cache.version>
+        <carbon.deployment.version>5.0.0</carbon.deployment.version>
+        <carbon.datasources.version>1.0.0</carbon.datasources.version>
+    </properties>
+
+    <dependencies>
+        <!-- Carbon -->
```

Comment:
Why do we need carbon dependencies here? The bundle should be copied to the dropins folder in the carbon kernel manually by the user right?